### PR TITLE
adding build for blacksheep-outliers

### PR DESCRIPTION
**Pull request by liliblu**
Hello, this is my first time trying to contribute to bioconda. I read the guidelines, but i'm new to github and i'm not sure i'm doing everything correctly, sorry if I am not. 
@
Here is the ping, since I am not a bioconda member: @bioconda/core

Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dpryan79** [comment]
FYI, the github repository doesn't seem to exist yet.

File: recipes/blacksheep-outliers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - matplotlib >=3.1.0
+    - numpy >=1.16.4
+    - pandas >=0.24.2
+    - pip >= 19.1.1
+    - python >= 3.7
```
Comment (dpryan79) [inline recipes/blacksheep-outliers/meta.yaml]: This doesn't work in python 3.6? You can remove the version specifier for pip.

**Comment by liliblu** [comment]
Hi thanks for getting to this so quickly, the repo is private right now, but I will fix that soon. 
I removed the version for pip and changed it to python 3.6, it should work for that as well.

**Comment by dpryan79** [comment]
Shall I add you to the bioconda team once this is merged? You can maintain the blacksheep recipe easier then. BTW, I see there's an R version, are you planning to add that too? I'm always looking for useful outlier detection methods :)

**Comment by liliblu** [comment]
Yes adding me would be great, thank you! There is an R version, I couldn't figure out how to add them both in the same package, is that the way to do it? Could you point me towards an example of that? Or should it be two different packages?
I hope this tool will be useful! It's for comparing groups of samples using outlier values, not for finding outlier samples, in case that's what you meant :-)

**Comment by dpryan79** [comment]
It's better to keep them split. Are you planning on adding the R package to CRAN or bioconductor?

Yeah, I meant finding outliers within groups, but always good to know about new packages either way.

**Comment by liliblu** [comment]
Cool, 
Yeah i think bioconductor is the goal, my collaborator @mcornwell1957 is making that version.

**Comment by liliblu** [comment]
Hi again, it looks like all the tests passed, please let me know if there is anything else I should do, other than make the github repo public. 
Thanks again!

**Comment by dpryan79** [comment]
Great, I made a small tweak to minimize things a bit and then I'll hit merge. You should receive your invite from github shortly.

Have @mcornwell1957 ping me when he's got a PR ready. If this is targeted toward bioconductor then we can either start out assuming it'll get accepted there (so calling the package bioconductor-blacksheep or something like that) or start it out as r-blacksheep (or r-blacksheep-outliers) and then just remove that recipe once it's in bioconductor.

**Comment by dpryan79** [comment]
Thanks @liliblu

**Comment by liliblu** [comment]
Thank you!

**Pull request merged**